.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+-   repo: git://github.com/pre-commit/pre-commit-hooks
+    sha: 29bf11d13689a0a9a895c41eb3591c7e942d377d
+    hooks:
+    -   id: autopep8-wrapper
+    -   id: check-json
+    -   id: check-yaml
+    -   id: check-xml
+    -   id: debug-statements
+    -   id: flake8
+    -   id: trailing-whitespace
```

Comment:
Added end-of-file-fixer, that looks like a good first set.  I'm really reaching for PHP, Perl, and C# hooks at this point, that's where most of the repo's code is.